config/default.yml

```diff
@@ -7,11 +7,14 @@ Bundler/OrderedGems:
   Enabled: false
 
 Style/StringLiterals:
-  Enabled: false
+  Enabled: true
```

Comment:
I'm fine with this in principle, but enabling it at this point (when projects have years of unenforced history) seems like it has the potential to create a lot of diff noise in PRs. I wouldn't say I feel strongly about it, but I'm not sure it's worth the hassle.